### ISSUE TITLE
[Bug]: Source Switzerland ZKRI not working

### ISSUE THREAD

**Issue by blurap** [mampfes/hacs_waste_collection_schedule#2561]
### I Have A Problem With:

A specific source

### What's Your Problem

Hello

First of all, thanks for this cool Integration!

The source sammelkalender.ch doesn't work for service provider zkri. I have tried to implement this source with all combinations and variations of possible values in the fields municipality and street. And in our region, there aren't many. ;-)

https://daten.zkri.ch/web/Sammelkalender/sammelkalender.html

I'd like to use the calendar for "Gemeinde..." = "Arth" and "Gebiet..." = "Goldau". It is not on street level.

Example paramenters form documentation did not work either.

Thanks for your work and your support.

### Source (if relevant)

sammelkalender_ch.md

### Logs

```Shell
no relevant logs
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
  - name: sammelkalender_ch
    args:
      service_provider: zkri
      municipality: Goldau
      street: adress
      hnr: 11
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by Andurilll** [comment]
Maybe delete the adress Herr Flower ;)

**Comment by 5ila5** [comment]
Street doubles as street/Region depending on what the form asks for, so as the form asks fro your `municipality` = Arth and then for a region (parameter name is `street`) Goldau and not for any house number you need to use:

```yaml
waste_collection_schedule:
  sources:
  - name: sammelkalender_ch
    args:
      service_provider: zkri
      municipality: Arth
      street: Goldau
```

The error message in the config flow GUI (or in your logs when using YAML configuration) should prompt you for the correct values. e.g. your config produces the error     `ValueError: Invalid municipality: Goldau, use one of ['Arth', 'Gersau', 'Ingenbohl', 'Küssnacht', 'Lauerz', 'Morschach', 'Muotathal', 'Rothenthurm', 'Sattel', 'Schwyz', 'Steinen', 'Steinerberg']`

and if you then update your municipality you should see `ValueError: Invalid street: Rossbergstrasse, use one of ['Arth (inkl. Oberarth)', 'Goldau']
`
hnr will just be ignored as it is not needed here


- If it still does not work make sure you're using the master version as the latest version 2.1.0 does not include this source yet. You can switch to the master version by (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

If you still encounter any errors please take a look at your logs and share them here.

**Comment by blurap** [comment]
> If it is your Adress, , maybe delete it. Herr Flower ;)

Thanks for the hint. Done!

**Comment by blurap** [comment]
Thank you for the quick assistance. I had already tried the suggested configuration before opening this ticket, but unfortunately without success. I have now tried it again, but still without success.

Additionally, as recommended, I attempted to install the master version. Unfortunately, I wasn't able to figure out which version is the master. Sorry about that. I ended up installing version 1.49, but the calendar still isn't displaying.

My current configuration is as follows:

```
waste_collection_schedule:
  sources:
  - name: sammelkalender_ch
    args:
      service_provider: zkri
      municipality: Arth
      street: Goldau
```

The following entry is visible in the log:
```
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: waste_collection_schedule.source_shell
Quelle: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:208
Integration: waste_collection_schedule (Dokumentation)
Erstmals aufgetreten: 18:16:59 (1 Vorkommnisse)
Zuletzt protokolliert: 18:16:59
source not found: sammelkalender_ch
```

I suspect that I have not yet installed the master version, and that doing so might resolve the issue. But how do I find the master version?

Thank you in advance for your continued assistance.

**Comment by 5ila5** [comment]
HACS -> WCS -> 3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by 5ila5** [comment]
They published HACS 2.0 last week. Looks like they removed this option from there. Not quite sure why and if this is really meant to be that way. Someone opened an issue there, but I do not quite get the answer (https://github.com/hacs/integration/issues/4009)

Looks like the way for HACS 2.0 is to: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA

**Comment by blurap** [comment]
That worked! Thank you!
